### PR TITLE
[Account Permissions] Removing all permissions to default users

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This PR is introducing new back end configs that prevent new users (default users) to access unwanted APIs.

Note that we can also manage users, tokens, groups and permissions through the django admin console by using this link : http://localhost:8000/admin/

* You need a super user account in order to access this console

image

## Type of change

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**API access while not authenticated:**

image

**API access while being authenticated as default/new users:**

image

**API access while being authenticated as super user:**

image

**Trying to access the users list with a default user:**

image

****Trying to access the users list with a super user:**

image

# Testing

Manual steps to reproduce this functionality:

1.  Try to access vulnerable APIs with a default user and make sure that you get the "You do not have permission to perform this action." error.
2.  Then, make sure that you can access the vulnerable APIs by using the super user account.

# Checklist

Applicable for all code changes.

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [ ] ~~My changes look good on IE 11+ and Chrome~~

### CODE REVIEW

File: backend/config/settings/base.py [hunk 1]
```diff
@@ -91,9 +91,8 @@
         "rest_framework.permissions.IsAuthenticatedOrReadOnly",
     ),
     "DEFAULT_AUTHENTICATION_CLASSES": (
-        "rest_framework_jwt.authentication.JSONWebTokenAuthentication",
```
Comment (caleybrock) [inline backend/config/settings/base.py]: Why no more JWT?

File: backend/config/urls.py [hunk 2]
```diff
@@ -18,19 +15,7 @@
...
-
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-
-# Browsable API login
-urlpatterns += [
-    url(r"^api/auth/", include("rest_framework.urls", namespace="rest_framework"))
-]
-
-if settings.DEBUG:
```
Comment (caleybrock) [inline backend/config/urls.py]: Do we not want the debug tools anymore?

File: backend/config/settings/base.py (hunk 1 above)
Comment (fnormand01) [reply backend/config/settings/base.py]: Good catch! I tried so many different configs, that I didn't remember that I deleted it. I put it back.
See commit #https://github.com/code-for-canada/project-thundercat/pull/308/commits/5c358f2235c97f6bdbdbacf918350f38996507dc

File: backend/config/urls.py (hunk 2 above)
Comment (fnormand01) [reply backend/config/urls.py]: I thought we weren't using it, but I tested it and it's very useful. Good catch again!
See commit #https://github.com/code-for-canada/project-thundercat/pull/308/commits/c1ef6926cea5e84ffe168323367ebb816dcd0ac4

**Pull request merged**